.github/workflows/ci.yml

```diff
@@ -35,9 +35,9 @@ jobs:
       - name: Run CI script
         # The script also runs golang-ci but it's ok: doesn't take too long and may be useful
         # to keep its linter errors in this log, too.
-        run: ./scripts/travis-ci.sh ci
+        run: ./scripts/github-ci.sh ci
         env:
-          TRAVIS_OS_NAME: linux
+          GITHUB_OS_NAME: linux
```

Comment:
TRAVIS_OS_NAME is a real env var in travis. GITHUB_OS_NAME is not a real env var in GH CI. Maybe best to rename it to simply OS_NAME or CI_OS_NAME.